spring-xd-shell/src/main/java/org/springframework/xd/shell/command/JobCommands.java

```diff
@@ -358,6 +360,9 @@ public String deployJob(
 				}
 				propertiesToUse = DeploymentPropertiesFormat.convert(props);
 				break;
+			case -1: // Neither option specified
+				propertiesToUse = Collections.EMPTY_MAP;
```

Comment:
Needs to be `propertiesToUse = Collections.<String, String> emptyMap();` for type safety.

IDEA doesn't seem to be as good as eclipse at flagging such things :smile: 

Will fix during merge.